hedvalidation/hedvalidation/hed_input_reader.py

```diff
@@ -367,7 +370,7 @@ def _validate_tag_levels_in_hed_string(self, hed_string_delimiter):
         validation_issues += self._tag_validator.run_tag_level_validators(top_level_tags, formatted_top_level_tags);
         return validation_issues;
 
-    def _validate_top_level_in_hed_string(self, hed_string_delimiter):
+    def _validate_top_level_in_hed_string(self, hed_string_delimiter):  #modify based on validator
```

Comment:
Not yet done?

Reply:
I don't believe that that was me. Does something seem wrong with the function?